packages/commonwealth/server/migrations/20230531190846-fix-old-polls.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.sequelize.query(`
+      UPDATE "Threads"
+      SET "has_poll" = CASE WHEN (
+        SELECT COUNT(*)
+        FROM "Polls"
+        WHERE "Threads"."id" = "Polls"."thread_id"
+      ) > 0 THEN TRUE ELSE FALSE END
+    `);
+  },
+
+  down: async (queryInterface, Sequelize) => {
+    await queryInterface.sequelize.query(
+      'UPDATE "Threads" SET "has_poll" = NULL'
```

Comment:
Is `has_poll` always NULL currently?

Reply:
cc @alexyoung23j

Reply:
No, this will work but I think you're right its better if its set to false. Updating.

Reply:
@alexyoung23j I more meant, why are we setting to NULL in down migration? Shouldn't we do nothing?

Reply:
Ah I see. Well, its just a weird one because there isn't a way to return to the previous state. I guess it's probably better to just do nothing yeah.